x-pack/plugins/encrypted_saved_objects/server/crypto/encrypted_saved_objects_service.ts

```diff
@@ -62,14 +62,12 @@ export class EncryptedSavedObjectsService {
 
   /**
    * @param encryptionKey The key used to encrypt and decrypt saved objects attributes.
-   * @param knownTypes The list of all known saved object types.
-   * @param log Ordinary logger instance.
+   * @param logger Ordinary logger instance.
    * @param audit Audit logger instance.
    */
   constructor(
     encryptionKey: string,
-    private readonly knownTypes: readonly string[],
```

Comment:
__note:__ this check didn't bring lots of value and it doesn't fit very well to NP since plugins will register their types during `setup` that would make order of these operations important. So since SO client will return error for not known type anyway I decided to ditch this check. What do you think?

Reply:
That makes sense to me. Since we're relying on specific behavior in a dependency for this functionality, what do you think about adding a functional test [here](https://github.com/elastic/kibana/blob/5fa99d47ee88a6a5e8f8619875c29c66f193574b/x-pack/test/plugin_api_integration/test_suites/encrypted_saved_objects/encrypted_saved_objects_api.ts#L11) to ensure that the public encryption/decryption endpoints behave correctly when this happens?

Reply:
Somehow missed that comment, sorry. Yeah, it's a great idea, will add this test in a follow-up.